Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -0,0 +1,53 @@
+import UIKit
+
+/// Renders a receipt in an AirPrint enabled printer.
+/// To be properly implemented in https://github.com/woocommerce/woocommerce-ios/issues/3978
+final class ReceiptRenderer: UIPrintPageRenderer {
+    private let lines: [ReceiptLineItem]
+    private let paymentIntent: PaymentIntent
+
+    private let attributes: [NSAttributedString.Key: Any] = [.font: UIFont(name: "HelveticaNeue", size: 48) as Any]
+
+    init(content: ReceiptContent) {
+        self.lines = content.lineItems
+        self.paymentIntent = content.paymentIntent
```

Comment:
I'm not sure receipts should know about paymentIntents. Eventually we will need to support receipts for other means of payment.

Have you considered passing structured data instead to the renderer? i.e. arrays of key:values that should be output in each section?

Reply:
Nevermind. I just saw your related comment below that this will change.

Reply:
Yep. It's changed in #4010